src/templates/next/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,14 @@
+import { ParagraphProps } from "~/common"
+import { getSanitizedInlineContent } from "~/utils/getSanitizedInlineContent"
+
+const Paragraph = ({ content }: ParagraphProps) => {
+  const sanitizedContent = getSanitizedInlineContent(content)
+  return (
+    <p
+      className="[&_a]:underline [&_a]:text-blue-500 hover:[&_a]:text-blue-700"
+      dangerouslySetInnerHTML={{ __html: sanitizedContent }}
```

Comment:
This is an isolated component to represent a paragraph node. It is still possible for users to put in weird illegal stuff that violates the purpose of this particular node, so I'm being explicit here on what is allowed and what isn't.

Hence, the sanitisation is 2-fold:
1. Sanitise the input content to only allow inline elements and not block elements.
2. Sanitise the input content to prevent illegal content that can cause XSS.

The second one is required here because we are doing an explicit `dangerouslySetInnerHTML`. The inputs from the user will likely be HTML content, **not** just text.

Reply:
Following up on this even though it's merged - can we omit the sanitisation and just pass the string as is **as the child**? 

Idt anything will happen tbh, just want to confirm behaviour